README.md

```diff
@@ -137,7 +137,7 @@ Make sure to select the right kernel (`conda_spam`) when running the jupyter not
 </details>
 
 Then in the browser tab that opens, navigate to a `.ipynb` file you would like
-to run &mdash; such as `01_spam_tutorial.ipynb` &mdash; and execute the
+to run &mdash; such as `spam/01_spam_tutorial.ipynb` &mdash; and execute the
```

Comment:
the instructions say to run `jupyter notebook spam`, so it will just be the filename